app/jobs/reports/region_cache_warmer_job.rb

```diff
@@ -0,0 +1,33 @@
+class Reports::RegionCacheWarmerJob
+  include Sidekiq::Worker
+  include Sidekiq::Throttled::Worker
+
+  sidekiq_options queue: :default
+  sidekiq_throttle(
+    concurrency: {limit: 1}
```

Comment:
Holupaminute. Was this revert intentional? https://github.com/simpledotorg/simple-server/pull/4330/commits/3f899f82a99a575e2164e0264154a2fcb107c4a6

I think it's fine though. Keep the concurrency at 1 first, make sure the job works well, then crank it up.

Reply:
Accidental change, just pushed ba0ed099d. We'll start at 2 and then crank up. Keeping it at 1 won't give us any benefits, so.